batchlib/analysis/cell_analysis_qc.py

```diff
@@ -139,6 +139,10 @@ def __init__(self,
                          marker_bg_key=marker_bg_key,
                          **super_kwargs)
 
+    def validate_input(self, path):
```

Comment:
Is this is still necessary? If yes, we should fix it!

Reply:
I will look at it now